lib/oidc/endpoints/authorize.ts

```diff
@@ -56,7 +57,7 @@ export function convertTokenParamsToOAuthParams(tokenParams: TokenParams) {
   if (tokenParams.responseType!.indexOf('id_token') !== -1 &&
     tokenParams.scopes!.indexOf('openid') === -1) {
     throw new AuthSdkError('openid scope must be specified in the scopes argument when requesting an id_token');
-  } else {
+  } else if (tokenParams.scopes) {
```

Comment:
Why this change? Does empty scope query parameter have a special meaning to the authorize endpoint?

Reply:
For authenticator enrolment via authorize endpoint `scope` should be deleted.
https://github.com/okta/okta-auth-js/pull/1324/files#diff-c7c807658c1f989f83fd7287c01e1c459ffa33b5afc14bb32dc9bd23b51e0048R39